core-services/image-mirroring/openshift/_config.yaml

```diff
@@ -40,9 +40,25 @@ images:
   - "4.17"
   - "4.17.0"
   - latest
+  "sriov-4.17":
```

Comment:
why do we need these `sriov` specific tags?

Reply:
because we can't have this as part of the 4.17 ImageStream as it's core OCP payload